style.css

```diff
@@ -1,129 +1,239 @@
-*{
-    margin: 0;
-    padding: 0;
-    
-}
-.header{
-    min-height:  100vh;
-    width: 100%; 
-    background-image: linear-gradient(rgba(4, 9, 30, 0.7),rgba(4, 9, 30, 0.7)),url(Images/banner.png);
-    
-    background-position: center;
-    background-size: cover;
-    position: relative;   
-    
-}
-nav{
-    
-    padding: 2% 3%;  
-    display: flex;   
-    justify-content: space-between;
-    align-items: center;    
-}
-nav img{
-    width: 150px;  
-}
-.nav-links{
-    flex: 1;
-    text-align: right;  
-    font-family: 'Poppins', sans-serif;    
-    
-}
-.nav-links ul li{
-    list-style: none;
-    display: inline-block;
-    padding: 8px 12px;
-    position: relative;   
+* {
+  margin: 0;
+  padding: 0;
+  font-family: BlinkMacSystemFont, -apple-system, Segoe UI, Roboto, Helvetica, Arial, sans-serif;
 }
 
-.nav-links ul li a{
-    color: #fff;
-    text-decoration: none;
-    font-size: 14px;
+.header {
+  min-height: 100vh;
+  width: 100%;
+  background-image: linear-gradient(rgba(4, 9, 30, 0.7), rgba(4, 9, 30, 0.7)), url(images/banner.png);
+  background-position: center;
+  background-size: cover;
+  position: relative;
 }
-.nav-links ul li::after{
-    content: '';
-    width: 0%;
-    height: 2px;
-    background: #f44336;
+
+.navbar {
+  padding: 2% 3%;
```

Comment:
I'd suggest decide on units you use for spacing on your page and be consistent about them, i.e. use offset values multiple to 10px: 10px, 20px, 20px etc. this makes layout look more consistent, especially when it comes to vertical offsets.

however, if you make a decision to use different unit - it's fine as long as you acknowledge that design might not me consistent for the element you style, and it's okay in this particular case